cmd/kubedrainer/serve.go

```diff
@@ -201,7 +201,7 @@ func GetNodeInformation(nodeName string, kubernetesClient k8s.Interface) (string
 	}
 	switch providerName {
 	case "aws":
-		awsNodeIDExpression := `^/(?P<Region>[a-zA-Z0-9-]+)[a-z]/(?P<InstanceID>[a-zA-Z0-9-]+)$`
+		awsNodeIDExpression := `^(?P<Region>[a-zA-Z0-9-]+)[a-z]\/(?P<InstanceID>[a-zA-Z0-9-]+)$`
```

Comment:
This will break backwards compatibility, but making the first `/` optional should make it work for both cases:
`^/?(?P<Region>[a-zA-Z0-9-]+)[a-z]/(?P<InstanceID>[a-zA-Z0-9-]+)$`

Reply:
Thanks for catching this.  I updated this as I had been checking with the wrong language.